src/test/resources/pages/instructorCommentsPageDeleteFrc.html

```diff
@@ -1744,9 +1778,9 @@ <h4 class="col-sm-9 text-color-primary">
             <div class="panel panel-info feedback-question-panel">
               <div class="panel-heading">
                 <b>
-                  Question 1
+                  Question 2
```

Comment:
The first question seems to get removed. I suspect it's an off-by-one error somewhere?